practice/00-HTML-CSS-basics/multimedia.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Showing multimedia</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+  <body>
+    <img src="media/Pierre-Auguste_Renoir_072.jpg" width="400" height="482" alt="Pierre Renoir - Jeune fille coiffant ses cheveux"/>
+    <audio src="media/CowboyBebop-GoodnightJulia.mp3" controls preload></audio>
+    <video src="media/Malosetti-inMemoriamFacundoPereyra.MOV" width="640" height="360" controls preload></video>
```

Comment:
All the media src should be relative urls ex: http://myalbum.com/photo.jpg, same goes for audio and video